### ISSUE TITLE
Shortcut `Control+I` and `Control+O` don't work on macOS Big Sure

### ISSUE THREAD

**Issue by kolya182** [bitcoin-core/gui#370]
**Expected behavior**

Using shortcut `Control+C` and `Control +O` from main screen and from `Node window` (called via `Control+I` as an example) to open Debug log file(`Control +O`) and Console `Control+C`

**Actual behavior**

To open Debug log file(`Control +O`) and  Console `Control+C`
`Control+I` `Control+N` `Control+P` work as expected.

**System information**
macOs Big Sure 11.4 Bitcoin Core 0.21.1

**Comment by hebasto** [comment]
> **Expected behavior**
> 
> Using shortcut `Control+C` and `Control +O` from main screen and from `Node window` (called via `Control+I` as an example) to open Debug log file(`Control +O`) and Console `Control+C`

I think your expectations are wrong. The actual shortcuts on macOS are as pictured on the following screenshot:

![Screenshot from 2021-07-04 10-49-26](https://user-images.githubusercontent.com/32963518/124377703-f30f7580-dcb5-11eb-9aaa-8604c6f90185.png)

Please note, that ⌘ is the [Command key](https://en.wikipedia.org/wiki/Command_key). If you are using non-Apple keyboard, it could be mapped to the [Windows key](https://en.wikipedia.org/wiki/Windows_key).

**Comment by jarolrod** [comment]
@kolya182 did this solve your issue: https://github.com/bitcoin-core/gui/issues/370#issuecomment-873541751

The commands are prefixed with command, not control, for macOS

**Comment by kolya182** [comment]
> > **Expected behavior**
> > Using shortcut `Control+C` and `Control +O` from main screen and from `Node window` (called via `Control+I` as an example) to open Debug log file(`Control +O`) and Console `Control+C`
> 
> I think your expectations are wrong. The actual shortcuts on macOS are as pictured on the following screenshot:
> 
> ![Screenshot from 2021-07-04 10-49-26](https://user-images.githubusercontent.com/32963518/124377703-f30f7580-dcb5-11eb-9aaa-8604c6f90185.png)
> 
> Please note, that ⌘ is the [Command key](https://en.wikipedia.org/wiki/Command_key). If you are using non-Apple keyboard, it could be mapped to the [Windows key](https://en.wikipedia.org/wiki/Windows_key).
> 
> UPDATE.
> 
> Specifically to a mnemonic shortcut bound to the "Open" button (to open a debug log file) -- from Qt [docs](https://doc.qt.io/qt-5/qshortcut.html#details):
> 
> > On Mac, shortcuts are disabled by default.

Actually UI is misguiding, especially underscored capitalized first character points to that
<img width="1435" alt="Screen Shot 2021-07-05 at 9 27 10 PM" src="https://user-images.githubusercontent.com/3277207/124542316-087cc080-ddd8-11eb-9be2-c3f243e95aad.png">

**Comment by jarolrod** [comment]
@kolya182 the screenshot you posted is using the [Qt Fusion style](https://doc.qt.io/archives/qt-5.8/gallery-fusion.html) which we introduced in #177 because of visual bugs with the version of Qt we were using at the time, see: #136.

macOS disables mnemonics and so its style factory doesn't show them. The `Fusion` style is more of the `Linux` style, so the mnemonics are shown when using that style factory.

This was reverted in #246 after bumping our Qt version to `5.12.10`. The next Bitcoin Core release will not have these misleading underscore
